### PR TITLE
[Team 3] Fix treaties not being sent

### PR DESCRIPTION
**Pull request by SaraFFdez**
# Summary

We noticed our agents were not sending treaties so this is a PR to fix that (feel free to reject this PR if it is too late for this). Sorry for it being so late (exams 😢).

### CODE REVIEW

**Review by jzzheng22 — APPROVED** [review]
Should wait for approval from @Silvertrousers  too

**Review by Silvertrousers — APPROVED** [review]

**Pull request merged**
